hcipy/optics/glass.py

```diff
@@ -133,3 +133,57 @@ def _parse_sellmeier_glass_catalogue(catalogue_file):
         database[di[0].replace(" ", "")] = make_sellmeier_glass(1, di[1:4].astype('float'), di[4::].astype('float'))
 
     return database
+
+def fresnel_reflection_coefficients(n1, n2, angle_of_incidence):
+    '''Calculates the fresnel reflection amplitude coefficient for an interface.
+
+    Parameters
+    ----------
+    n1 : array_like
+        Refractive index of the first medium
+    n2 : array_like
+        Refractive index of the second medium
+    angle_of_incidence : array_like
+        The angle of incidence with respect to the normal of the interface.
+
+    Returns
+    -------
+    r_s, r_p
+        The reflection coefficients for s and p polarization.
+
+    '''
+    n_rel = n2 / n1
+    n_cos_theta_out = np.sqrt(n_rel**2 - np.sin(angle_of_incidence)**2)
```

Comment:
What happens past the critical angle? Does this give imaginary values? Does it give NaNs?